### ISSUE TITLE
Associations - support in CRUD screen templates

### ISSUE THREAD

**Issue by vyacheslav-pushkin** [Amplicode/amplicode-frontend#243]
Implement simplest solution that does't require user to specify addititional query. Can be read-only components.

✅ - already done
🚧 - to do

| Cardinality | Screen | Component |
| ----- | ---- | ---- |
| to-One | list | ✅ render display name use getXXXDisplayName function |
|  | editor | ✅ EntityLookupField |
|  | details | 🚧 render display name use getXXXDisplayName function |
| to-Many | list | render a list of display names using getXXXDisplayName functions  |
|  | editor | render a list of display names using getXXXDisplayName functions (**read-only component, will not be possible to edit**) |
|  | details | render a list of display names using getXXXDisplayName functions  |

For to-one associations already done (lookup field). Need to implement 

to-many, to-one.

**Comment by vadimbasko** [comment]
Progress on issue is suspended. For many-to-many association we need example in amplicode petclinic first.

**Comment by vyacheslav-pushkin** [comment]
[Example in amplicode petclinic](https://youtrack.haulmont.com/issue/AB-52) is implemented, issue unblocked
